geoportal/c2cgeoportal_geoportal/scaffolds/nondockercreate/geoportal/webpack.apps.js.mako_tmpl

```diff
@@ -65,4 +65,9 @@ module.exports = {
     rules
   },
   plugins: plugins,
+  resolve: {
+    alias: {
+      {{package}}: path.resolve(__dirname, '{{package}}_geoportal/static-ngeo/js'),
```

Comment:
Shouldn't it be `${package}` here ? (twice in the line)

Reply:
no, it should be interpreted when we instantiate the scaffold and in your project you will directly see 'sitn' :-)

Reply:
Oh OK, I see... It is because of the `tmpl`, you are right !  thanks !